src/telemetry/Manager.h

```diff
@@ -2,161 +2,62 @@
 
 #pragma once
 
+#include <prometheus/exposer.h>
+#include <prometheus/registry.h>
+#include <condition_variable>
 #include <cstdint>
 #include <initializer_list>
+#include <memory>
 #include <string_view>
-#include <unordered_map>
-#include <variant>
 #include <vector>
 
 #include "zeek/IntrusivePtr.h"
 #include "zeek/Span.h"
 #include "zeek/telemetry/Counter.h"
 #include "zeek/telemetry/Gauge.h"
 #include "zeek/telemetry/Histogram.h"
-
-#include "broker/telemetry/fwd.hh"
-
-namespace broker {
-class endpoint;
-}
+#include "zeek/telemetry/ProcessStats.h"
+#include "zeek/telemetry/Utils.h"
 
 namespace zeek {
 class RecordVal;
 using RecordValPtr = IntrusivePtr<RecordVal>;
 } // namespace zeek
 
-namespace zeek::Broker {
-class Manager;
-}
-
 namespace zeek::telemetry {
 
 /**
  * Manages a collection of metric families.
  */
-class Manager {
+class Manager final {
 public:
-    friend class Broker::Manager;
-
     Manager();
 
     Manager(const Manager&) = delete;
 
     Manager& operator=(const Manager&) = delete;
 
-    virtual ~Manager() = default;
+    ~Manager() = default;
 
     /**
      * Initialization of the manager. This is called late during Zeek's
      * initialization after any scripts are processed.
```

Comment:
Could add a one-liner here about what our implementation does: set up Civetweb and a few metrics.